pkg/util/source/inspector.go

```diff
@@ -35,13 +35,22 @@ var (
 	doubleQuotedTo   = regexp.MustCompile(`\.to\s*\(\s*"([a-zA-Z0-9-]+:[^"]+)"`)
 	doubleQuotedToD  = regexp.MustCompile(`\.toD\s*\(\s*"([a-zA-Z0-9-]+:[^"]+)"`)
 	doubleQuotedToF  = regexp.MustCompile(`\.toF\s*\(\s*"([a-zA-Z0-9-]+:[^"]+)"`)
+	languageRegexp   = regexp.MustCompile(`language\s*\(\s*["|']([a-zA-Z0-9-]+[^"|']+)["|']\s*,.*\)`)
 
 	additionalDependencies = map[string]string{
-		`.*JsonLibrary\.Jackson.*`:      "camel:jackson",
-		`.*\.hystrix().*`:               "camel:hystrix",
-		`.*restConfiguration().*`:       "camel:rest",
-		`.*rest(("[a-zA-Z0-9-/]+")*).*`: "camel:rest",
-		`^\s*rest\s*{.*`:                "camel:rest",
+		`.*JsonLibrary\.Jackson.*`:                        "camel:jackson",
+		`.*\.hystrix().*`:                                 "camel:hystrix",
+		`.*restConfiguration().*`:                         "camel:rest",
+		`.*rest(("[a-zA-Z0-9-/]+")*).*`:                   "camel:rest",
+		`^\s*rest\s*{.*`:                                  "camel:rest",
+		`.*\.groovy\s*\(.*\).*`:                           "camel:groovy",
+		`.*\.(jsonpath|jsonpathWriteAsString)\s*\(.*\).*`: "camel:jsonpath",
+		`.*\.ognl\s*\(.*\).*`:                             "camel:ognl",
+		`.*\.mvel\s*\(.*\).*`:                             "camel:mvel",
+		`.*\.?simple\s*\(.*\).*`:                          "camel:bean",
+		`.*\.xquery\s*\(.*\).*`:                           "camel:saxon",
+		`.*\.xpath\s*\(.*\).*`:                            "camel:xpath",
```

Comment:
I think `xpath` exists also in a form of `transform(xpath(...))`

Reply:
Thanks, I pushed a fix + test for that.